cpp/tests/io/orc_test.cpp

```diff
@@ -2055,4 +2058,40 @@ TEST_F(OrcStatisticsTest, Empty)
   EXPECT_EQ(ts6.count[0], 0);
 }
 
+TEST_P(OrcCompressionTest, Basic)
+{
+  constexpr auto num_rows     = 12000;
+  auto const compression_type = GetParam();
+
+  // Generate compressible data
+  auto int_sequence =
+    cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i % 100; });
+  auto float_sequence =
+    cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i / 32; });
```

Comment:
Given that it's a `float_sequence`, should we consider the following?
```suggestion
    cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i / 32f; });
```

Reply:
Wait, no, hang on.  That's not the point of this test.